pkg/storegateway/bucket.go

```diff
@@ -67,6 +67,8 @@ const (
 	// Labels for metrics.
 	labelEncode = "encode"
 	labelDecode = "decode"
+
+	lazyLoadedFile = "lazy-loaded.proto"
```

Comment:
[nit] Should this be `lazy-loaded.pb`? `*.proto` is used for the text files that describe message types, whereas this file will contain protobuf-encoded data.

Reply:
I was not sure. But yes will just use .pb.